### PR TITLE
Added RobustScaler for feature scaling

### PR DESCRIPTION
**Pull request by csbrandt**
- scale training set using [RobustScaler](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.RobustScaler.html#sklearn.preprocessing.RobustScaler)
- apply the same scaling transformation to test set

### CODE REVIEW

**Comment by anfederico** [comment]
I'm going to incorporate this in the next major update (coming soon). I'll add you to the contributors list.
